plugin.video.fullycharged/addon.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="plugin.video.fullycharged"
+	name="Fully Charged"
+	version="1.0.1"
+	provider-name="zag">
+  <requires>
+    <import addon="xbmc.python" version="2.1.0"/>
```

Comment:
Thanks, yes Tamland suggested moving icon.png and fanart.png to the root and modifying the asset tag accordingly. This maintains backward compatibility. I'll change the xbmc.python to 2.14.0 and branch to Gotham.